src/style.css

```diff
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/198086481-c1a1ba5f-c7ee-4166-af53-009713ca7be3.png)
and better use one-direction margins. for example margin-right: 4px.

Reply:
Thank you! Changed styles but tests failed :( What can be the reason?